scripts/nrf_profiler/README.rst

```diff
@@ -137,6 +137,81 @@ You can use it  for visualization or calculating statistics.
    An application must provide its own implementation.
    See the :ref:`nrf_desktop_nrf_profiler_sync` for an example implementation used by the :ref:`nrf_desktop` application.
 
+.. _nrf_profiler_script_calculating_statistics:
+
+Calculating statistics
+======================
+
+The :file:`calc_stats.py` script is used to calculate and visualize the nRF Profiler event propagation statistics (statistics for time intervals between nRF Profiler events) for a given dataset.
+A JSON file containing test presets specifies the events to be analyzed.
+The script calculates and displays the following statistics for each test preset:
+
+* basic statistics (min, max, mean, median, standard deviation)
+* histogram plot
+* plot of subsequent time intervals between events
+
+For some of the statistics, the script also filters out time intervals that deviate from the other records.
+These statistics are denoted by a ``no outliers`` suffix in the plot title and output file name.
+
+Apart from displaying the statistics, the scipts also stores them as local files under the :file:`data_stats/` directory.
+The results are organized in subdirectories based on the dataset name, time range, and test preset name.
+
+When running the script from the command line, provide the dataset name (``test1``) and path to the JSON file containing test presets (:file:`stats_nordic_presets/app_event_manager_profiler_tracer.json`).
```

Comment:
Please mention that the user can also use regular Profiler sample - event manager is not required.

Reply:
Added:
`You can use the script to calculate statistics for custom nRF Profiler events or Application Event Manager events, or both.`